README.rst

```diff
@@ -77,86 +64,46 @@ any code that was using the old version of pymedphys. We try to be abreast of
 this by providing details of any breaking changes from one release to the next
 within the `Release Notes`_.
```

Comment:
The following link is broken:

![image](https://user-images.githubusercontent.com/6559099/105962341-ce461f00-60d3-11eb-80ec-ac4e2664110c.png)

Reply:
I went to dive in and fix this... but the release notes aren't actually within the navigation anymore?

Reply:
Sorry about that. Back in. I left it in its own title-less group above the others on the sidedbar. It didn't seem to obviously fir under any of the others to me. I have no real preference: what do you think?